### DISCUSSION TITLE
Accessibility with NextJS

### DISCUSSION THREAD

**Discussion by RidgewellServices** [vercel/next.js#12424] [Help] [answered]
How is accessibility meant to be implemented in Next? Do I need to create a Custom Document to do such things as skip to main?

**Comment by leerob** [comment, marked as answer]
Since Next.js is a framework for React, you can utilize a11y resources already created for React. You could override `_app.js` to have a shared layout that contains a skip nav.

https://reactjs.org/docs/accessibility.html
https://reacttraining.com/reach-ui/skip-nav/

**Reply by RidgewellServices** [reply]
Thank you. I was so focused on NextJS, I didn't think to look at the React docs. I think this will give me what I need.

**Reply by RidgewellServices** [reply]
Thanks again, leerob. I got it all working.

**Reply by leerob** [reply]
Awesome! If you mark this as answered, it will help others browsing questions that need solved.

**Reply by RidgewellServices** [reply]
👍🏽
